hikari/impl/entity_factory.py

```diff
@@ -2423,11 +2425,20 @@ def deserialize_template(self, payload: data_binding.JSONObject) -> template_mod
 
     @staticmethod
     def _set_user_attributes(payload: data_binding.JSONObject) -> _UserFields:
+        accent_color: undefined.UndefinedOr[color_models.Color]
+
```

Comment:
This is probably the biggest nitpick
```suggestion
        accent_color: undefined.UndefinedOr[color_models.Color]
```

Reply:
I don't understand.

Reply:
Remove the empty line after accent_color :P

Reply:
github showed me that there was no change lmao